Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.11
+
+WORKDIR /cinema
+
+COPY requirements.txt .
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+RUN adduser --disabled-password --no-create-home my_user
+
+COPY . .
```

Comment:
Copying the entire context (`COPY . .`) can lead to larger image sizes if there are unnecessary files in the context. Consider using a `.dockerignore` file to exclude files that are not needed in the image.